style.css

```diff
@@ -0,0 +1,390 @@
+@font-face {
+  font-family: NexaLight;
+  src: url('./Font/NexaLight.otf');
```

Comment:
folder name should be "fonts" also male all folder name in second latter only not in capital @aeshapatel025

Reply:
changes done